types/react-native-scrollable-tab-view/index.d.ts

```diff
@@ -2,11 +2,12 @@
 // Project: https://github.com/brentvatne/react-native-scrollable-tab-view
```

Comment:
Please update the version number in the first line if these changes are specific to v0.10.

Reply:
okay.